src/shortcuts/__tests__/newThought.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { store } from '../../store'
+import { keyDown } from '../../shortcuts'
+import { mount } from 'enzyme'
+import { importText } from '../../action-creators'
+import { RANKED_ROOT, ROOT_TOKEN } from '../../constants'
+import { exportContext } from '../../selectors'
+
+/** Wrapper dummy component to simulate global key down events for shortcuts. */
+const TestWrapper = () => <div onKeyDown={keyDown}></div>
+
+it('empty thought should outdent when hit enter', async () => {
+
+  // skip tutorial and close welcome modal
+  await store.dispatch({ type: 'modalComplete', id: 'welcome' })
+  await store.dispatch({ type: 'tutorial', value: false })
+
+  // import thoughts
+  await store.dispatch(importText(RANKED_ROOT, `
+  - a
+    - b
+      - c
+        - d 
+          - e
+            - f`))
+
+  await store.dispatch({ type: 'setCursor', thoughtsRanked: [
+    { value: 'a', rank: '0' },
+    { value: 'b', rank: '1' },
+    { value: 'c', rank: '2' },
+    { value: 'd', rank: '3' },
+    { value: 'e', rank: '4' },
+    { value: 'f', rank: '5' },
+  ] })
+
+  // mount wrapper component to simulate global key down events
+  const wrapper = await mount(<TestWrapper/>)
```

Comment:
What about calling `keyDown` directly to avoid `mount`?

Reply:
`keyDown` uses synthesized event `e`. So we have to pass an object with properties such as `preventDefault`, `metaKey`. We can pass a noop function as `preventDefault` to avoid error. Should we do that ?

Reply:
Yes, I think that would be cleaner. We are then just mocking the event rather than the whole mounted node.